<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditorPageHUDWidgets</class>
 <widget class="QWidget" name="ScriptEditorPageHUDWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="list"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="buttonNew">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonMoveUp">
         <property name="text">
          <string>Move Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonMoveDown">
         <property name="text">
          <string>Move Down</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDelete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="position">
       <item>
        <property name="text">
         <string>Top Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Top Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Top Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Upper Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Upper Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Upper Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lower Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bottom Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bottom Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unknown 11 (Hidden?)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Position</string>
       </property>
       <property name="buddy">
        <cstring>position</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ImageStackWidget" name="preview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>450</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageStackWidget</class>
   <extends>QFrame</extends>
   <header>../generic/ImageStackWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>list</tabstop>
  <tabstop>buttonNew</tabstop>
  <tabstop>buttonMoveUp</tabstop>
  <tabstop>buttonMoveDown</tabstop>
  <tabstop>buttonDelete</tabstop>
  <tabstop>position</tabstop>
 </tabstops>
 <resources>
  <include location="../../Resources/main_window.qrc"/>
 </resources>
 <connections/>
</ui>
